modules/openapi-generator/src/main/resources/typescript-fetch/recordGeneric.mustache

```diff
@@ -0,0 +1,148 @@
+import {ApiRecordUtils, knownRecordFactories{{#isMetaDataResponse}}, appFromJS, NormalizedRecordEntities{{/isMetaDataResponse}}} from "../runtimeSagasAndRecords";
+import {List, Record, RecordOf} from 'immutable';
+import {Schema, schema, NormalizedSchema} from "normalizr";
+
+import { {{classname}} } from './';
+
+{{#hasEnums}}
+{{#vars}}
+{{#isEnum}}
+import { {{classname}}{{enumName}} } from "./";
+{{/isEnum}}
+{{/vars}}
+{{/hasEnums}}
+
+{{#hasImports}}
+import {
+    {{#imports}}
+    {{{.}}},
+    {{/imports}}
+{{#modelImports}}
+    {{.}}Record,
+    {{#lambda.camelcase}}{{.}}{{/lambda.camelcase}}RecordUtils,
+{{/modelImports}}
+} from './';
+{{/hasImports}}
+
+export const {{classname}}RecordProps = {
+	recType: "{{classname}}Record" as "{{classname}}Record",
+{{#vars}}
+    {{#isArray}}
+    {{#itemsAreModels}}
```

Comment:
This is directly related to what I have said above, you don't need these 3 new properties.
The logic is: 
* All items are models unless they are native type
* You can know that a model is an "entity" if isUnique is true.

Reply:
Will check if {{^nativeType}} is the same as itemsAreModels ...